libs/mve/image_io.cc

```diff
@@ -87,7 +87,9 @@ load_file (std::string const& filename)
         try
         {
             ImageHeaders header = load_mvei_file_headers(filename);
-            if (header.type != IMAGE_TYPE_UINT8) { throw; }
+            if (header.type != IMAGE_TYPE_UINT8) {
+                throw util::Exception("Invalid image format");
+            }
             ImageBase::Ptr image = load_mvei_file(filename);
             return std::static_pointer_cast<ByteImage>(image);
```

Comment:
Because the type is essentially checked manually before I thought it is unnecessary do another runtime check with `std::dynamic_pointer_cast`. However, this assumes that the file does not change between the `load_mvei_file_headers` and `load_mvei_file`. I guess the runtime check is preferable in this case.